src/modules/colorPicker/ColorPickerUI/App.xaml.cs

```diff
@@ -23,8 +24,16 @@ public partial class App : Application, IDisposable
         private bool disposedValue;
         private ThemeManager _themeManager;
 
+        private CancellationTokenSource NativeThreadCTS { get; set; }
+
+        [Export]
+        private static CancellationToken ExitToken { get; set; }
+
         protected override void OnStartup(StartupEventArgs e)
         {
+            NativeThreadCTS = new CancellationTokenSource();
+            ExitToken = NativeThreadCTS.Token;
```

Comment:
where is this ExitToken used?

Reply:
it's used in an overengineered way of constructing `MainViewModel` by employing assembly-level DI framework